sources/packages/web/src/components/generic/TitleValue.vue

```diff
@@ -0,0 +1,26 @@
+<!--
+Used when we need to display title and the value inside a container
+-->
+<template>
+  <v-row class="mt-1 mb-2 ml-0">
```

Comment:
`ml-0` really required here? same in line 8

Reply:
The ml-0 is adding a tab, as discussed.